src/richie/apps/core/context_processors.py

```diff
@@ -57,3 +60,21 @@ def site_metas(request):
     Context processor to add the current *Site* metas to the context
     """
     return get_site_metas(is_secure=request.is_secure())
+
+
+def frontend_context(request):
+    """
+    Context processor to add the context as expected by the frontend to the template context.
+    """
+    return {
+        "FRONTEND_CONTEXT": json.dumps(
+            {
+                "context": {
+                    "assets": {"icons": static("richie/icons.svg")},
+                    "environment": getattr(settings, "ENVIRONMENT", ""),
+                    "release": getattr(settings, "RELEASE", ""),
```

Comment:
Would it be possible to get the release from the frontend code? I'm thinking of someone that would customize the frontend and so run a different release number in the frontend and in the backend.

Reply:
Done 👍